lib/tasks/participa.rake

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+# Next tasks are necessary to be executed on production
+# because there are some records in the database that don't have
+# a component or participatory space related
+# This may be due to an incorrect import of the old system or to a bad delete
+# of the records.
+# At the request of the GENCAT, it has been decided not to delete these records.
+# If these tasks don't run before the deployment of 0.15 version,
+# there may be problems with some migrations, and they will fail.
+namespace :participa do
+  # Components ------
+  #
+  # Find the Organization.
+  # Find or Create an unpublished participatory process
+  # Takes all the components that don't have a participatory space related.
+  # Assign the created participatory process to the components
+  # without participatory space .
+  desc 'Fix Orphan Component records'
```

Comment:
```suggestion
  desc <<EODESC
Fix Orphan Component records:
- Find the Organization.
- Find or Create an unpublished participatory process
- Takes all the components that don't have a participatory space related.
- Assign the created participatory process to the components without participatory space.
EODESC
```

Reply:
I have followed the same description system as other rake tasks in Decidim.
As for example thsse one:
 https://github.com/decidim/decidim/blob/master/decidim-core/lib/tasks/decidim_metrics_tasks.rake 
or this https://github.com/decidim/decidim/blob/master/decidim-core/lib/tasks/decidim_data_portability_tasks.rake

If you prefer to use the own rake's system, I can change it.
What you prefer?